src/containers/JobDetails/JobDetails.test.js

```diff
@@ -45,47 +45,67 @@ describe('JobDetails', () => {
     render(
       <Provider store={ store }>
         <MemoryRouter>
-          <JobDetails data={store.jobInfo}  key={'12345'}/>
+          <JobDetails />
         </MemoryRouter>
       </Provider>
     )
 
     const backButton = screen.getByRole('button', {name: 'Back'});
-    const jobType = screen.getByText('Job Type: Labor & Materials');
-    const siteName = screen.getByText("Job Site Name: Home");
-    const siteContact = screen.getByText('Job Site Contact Name: Taryn');
+    const jobTypeLabel = screen.getByText('Job Type:');
+    const jobType = screen.getByText('Labor & Materials');
+    const siteNameLabel = screen.getByText("Job Site Name:");
+    const siteName = screen.getByText("Home");
+    const siteContactLabel = screen.getByText('Job Site Contact Name:');
+    const siteContact = screen.getByText('Taryn');
     const jobSiteAddress1 = screen.getByText('Job Site Address:', {exact: false});
     // const jobSiteAddress2 = screen.getByText("200 Washington St., ", { exact: false })
```

Comment:
I know I removed a lot of this from my file, I'd double check to see if there's a merge conflict